shub/apps/api/urls/collections.py

```diff
@@ -28,7 +28,8 @@ class CollectionSerializer(serializers.HyperlinkedModelSerializer):
 
     containers = serializers.SerializerMethodField("list_containers")
 
-    def list_containers(self, collection):
+    @staticmethod
+    def list_containers(collection):
```

Comment:
Did you use some linter to come up with these changes?

I wouldn't change a serializer method field, as it's expecting to have a self (and not be static) see https://www.django-rest-framework.org/api-guide/fields/#serializermethodfield. Did you test these API views with this change?